src/current/releases/cloud.md

```diff
@@ -523,7 +523,7 @@ For an in-depth explanation of CockroachDB {{ site.data.products.serverless }} p
 
 - The following features are now available for CockroachDB {{ site.data.products.serverless }} clusters running CockroachDB [v22.2.0]({% link releases/v22.2.md %}) or later:
   - [Query cancellation]({% link {{site.current_cloud_version}}/cancel-query.md %}#considerations) using the PostgreSQL wire protocol (pgwire).
-  - [`EXPLAIN ANALYZE`]({% link {{site.current_cloud_version}}/explain-analyze.md %}) now gives an estimate of the [Request Units (RUs)]({% link cockroachcloud/plan-your-cluster.md %}#request-units) a query consumes.
+  - [`EXPLAIN ANALYZE`]({% link {{site.current_cloud_version}}/explain-analyze.md %}) now gives an estimate of the [Request Units (RUs)]({% link cockroachcloud/plan-your-cluster-basic.md %}#request-units) a query consumes.
```

Comment:
I believe we should link to the standard page, sinc this note is about Cloud in general.

Reply:
RUs only remain for Basic, so I'm making an alternative edit to the bullet